.babelrc

```diff
@@ -1,9 +1,10 @@
 {
   "presets": [
     [
-      "babel-preset-env",
+      "@babel/preset-env",
       {
-        "modules": false
+        "modules": false,
+        "targets": { "ie": "10" }
```

Comment:
https://polyfill.io supports IE/8 which is an ES3 environment whereas IE/10 is an ES5 environment.
The main difference between those environments for a compiler like BabelJS is in ES5, keywords can be used as property-names via dot-notation whereas in ES3 they have to use bracket-notation.

E.G. ES5 can have this code and ES3 can not have this code: `var obj = {}; obj.delete = 1;`

Reply:
well spotted https://github.com/babel/babel/issues/6606 suggests to add babel-plugin-transform-es3-member-expression-literals 

also do you want "targets": { "ie": "8" } (apparently the plugin above is still needed even with this)